node/src/accountant/db_big_integer/big_int_db_processor.rs

```diff
@@ -408,61 +442,59 @@ impl StdNumParamFormNamed {
     }
 }
 
-pub struct Param<'a> {
-    value_pair: (&'a str, &'a dyn ExtendedParamsMarker),
-    participates_in_overflow_clause: bool,
+pub enum Param<'params> {
+    BothClauses(DisplayableRusqliteParamPair<'params>),
+    MainClauseLimited(DisplayableRusqliteParamPair<'params>),
```

Comment:
Maybe you can just say `MainClause` and `BothClauses` here. Also, if we can get rid of the lifetime, the `enum` will become easier to read.